src/main/kotlin/com/exactpro/th2/common/schema/message/impl/rabbitmq/transport/ParsedMessage.kt

```diff
@@ -161,14 +161,18 @@ class ParsedMessage private constructor(
     }
 }
 
+@Suppress("PropertyName")
 private sealed class BaseParsedBuilder<out T : ParsedMessage.Builder<T>> : ParsedMessage.Builder<T> {
     protected var idBuilder: MessageId.Builder? = null
     protected var id: MessageId? = MessageId.DEFAULT
-    protected var eventId: EventId? = null
+    protected var _eventId: EventId? = null
```

Comment:
You can just declare it as var here and make the setter private or protected
```suggestion
    override var eventId: EventId? = null
      private set
```
Should do the work